libs/platform/kernel/ebpf_platform_kernel.c

```diff
@@ -920,3 +925,36 @@ ebpf_utf8_string_to_unicode(_In_ const ebpf_utf8_string_t* input, _Outptr_ wchar
     ebpf_free(unicode_string);
     return retval;
 }
+
+intptr_t
+ebpf_platform_reference_process()
+{
+    PEPROCESS process = PsGetCurrentProcess();
+    ObReferenceObject(process);
+    return (intptr_t)process;
+}
+
+_Ret_maybenull_ ebpf_process_state_t*
```

Comment:
?? what is the need for this function?

Reply:
Since ebpf_platform.h only contains:
```typedef struct _ebpf_process_state ebpf_process_state_t;```
the file which includes this does not know the size of the struct -- so `sizeof(ebpf_process_state_t)` will give compiler error.